src/GitVersion.Core/VersionCalculation/BaseVersionCalculator.cs

```diff
@@ -58,11 +58,40 @@ public BaseVersion GetBaseVersion()
                 BaseVersion baseVersionWithOldestSource;
                 if (matchingVersionsOnceIncremented.Any())
                 {
-                    var oldest = matchingVersionsOnceIncremented.Aggregate((v1, v2) =>
-                        v1.Version!.BaseVersionSource!.When < v2.Version!.BaseVersionSource!.When ? v1 : v2);
-                    baseVersionWithOldestSource = oldest!.Version!;
-                    maxVersion = oldest;
-                    this.log.Info($"Found multiple base versions which will produce the same SemVer ({maxVersion.IncrementedVersion}), taking oldest source for commit counting ({baseVersionWithOldestSource!.Source})");
+                    if (matchingVersionsOnceIncremented.Count > 1)
+                    {
+                        log.Info($"Found multiple base versions which will produce the same SemVer ({maxVersion.IncrementedVersion})");
```

Comment:
I've also improved the logging here that I thought was misleading. It was initially saying that there were many base versions, when there were only 1.

So, if we don't get the logic change, I'll refactor this PR just to get the better logging in.